utils/errorHandlers/error.handler.ts

```
@@ -0,0 +1,54 @@
+        originalRequest._retry = true
+        try {
+          await refreshToken()
+          originalRequest.headers.Authorization = `Bearer ${authStore.getToken()}`
+          return api.request(originalRequest)
+        } catch (refreshError) {
+          authStore.resetAuth()
+          const statusMessage = errorNames[name] || errorMessages[401]
+          throw createError({ statusCode: 401, statusMessage })
+        }
+      }
+      const statusMessage401 = errorNames[name] || errorMessages[401]
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)